### DISCUSSION TITLE
如图控件用WPF该怎么做？？

### DISCUSSION THREAD

**Discussion by yoi102** [BYJRK/DotNet-Discussions#142] [Q&A]
请问用wpf怎么做出这样的控件吗？    
或者能提供思路吗？

![c](https://github.com/BYJRK/DotNet-Discussions/assets/77535233/bf3d50ef-4120-428a-8bec-89afa3661f6f)

**Comment by yoi102** [comment]
https://github.com/HandyOrg/HandyControl

**Reply by BYJRK** [reply]
HandyControl 就有这样的控件

https://handyorg.github.io/handycontrol/extend_controls/compareSlider/
